pkg/projectfile/projectfile.go

```diff
@@ -1273,9 +1273,9 @@ func GetProjectPaths(cfg ConfigGetter, namespace string) []string {
 	return paths
 }
 
-// storeProjectMapping associates the namespace with the project
+// StoreProjectMapping associates the namespace with the project
 // path in the config
-func storeProjectMapping(cfg ConfigGetter, namespace, projectPath string) {
+func StoreProjectMapping(cfg ConfigGetter, namespace, projectPath string) {
```

Comment:
This doesn't seem to be necessary for this PR.

Reply:
Push isn't storing a project entry for a new project. At least it wasn't happening during my testing. Only when the project was activated did the config change. So I exposed this function so push could update the config when converting a headless project. If we don't want that behaviour in push I can revert this.